package.json

```
@@ -51,12 +51,14 @@
     "ferrum": "1.8.0",
     "lodash.pick": "4.4.0",
     "lodash.pickby": "4.6.0",
+    "minimatch": "^3.0.4",
```

```suggestion
    "minimatch": "3.0.4",
```

tip: use `npm add -E minimatch` to add a pinned dependency